added wave sort algorithm

aryan13072001 opened the pull request:
I have added wave sort algorithm in c++.Please merge my pr and add hactoberfest label on it.

aryan13072001 committed:
added wave sort algorithm

Ashad001 reviewed (changes requested):
Sorry, but the array is not been sorted. Please make changes and try again.
P.S- A little description of your algorithm would be appreciated

aryan13072001 committed:
Update wavesort.cpp

aryan13072001 committed:
updated wavesort.cpp

Wavesort algorithm is used for sorting an unsorted array in such a way that the elements of the array gets arranged in a manner,lets say,that the first element is lesser than the second element,third element is greater than second but lesser than fourth one and so on,so as to form a wave form type of array.It is represented as 
arr[0] >= arr[1] <= arr[2] >= arr[3] <= arr[4] >= …..

Ashad001 reviewed (approved):
Thank you for contributing, and keep it up.
Please leave a star on this repo!

aryan13072001 commented:
can you please add the hactoberfest label on it.
